### ISSUE TITLE
V1.1.8: Strange warning

### ISSUE THREAD

**Issue by erkr** [music-assistant/hass-music-assistant#196]
Maybe not relevant, but smells
```
Logger: music_assistant.mass.music
Source: /usr/local/lib/python3.9/site-packages/music_assistant/controllers/music/__init__.py:110 
First occurred: 20:35:09 (2 occurrences) 
Last logged: 20:35:09

Provider ProviderType.DATABASE is not available

```

**Comment by marcelveldt** [comment]
Isn't there some more logging surrounding it ? Maybe in the full log (homeassistant.log) ?

**Comment by erkr** [comment]
No, but debug logging is not on. I will switch it on and check if it happens again

**Comment by marcelveldt** [comment]
No debug logging is not needed.

And nvm, already found the problem. Small typo.

**Comment by marcelveldt** [comment]
released small patch release v1.1.9 which contains the fix (no rescan needed)

**Comment by marcelveldt** [comment]
Couldn't reproduce this on v1.1.14 I will do a bit more testing tomorrow but it might be resolved due to the final db changes I made today

**Comment by erkr** [comment]
I have seen them a few times already just by adding tracks to the queue (functionally i noticed no impact)

**Comment by erkr** [comment]
With v1.1.15 I still see them now and then when browsing the collection

**Comment by erkr** [comment]
Hi Marcel
The pitty: This warning still occurs in v1.1.16
The good: I found a simple way to reproduce the warning. 

The warning is triggered when opening the 'show info' menu for  particular tracks. BUT it seemed to not reproduce:

- it only triggers once per track! (when a background task starts - spinning wheel in the corner)
- and typically only for the first track that I open after opening MA.

Hopefully this helps in finding the trigger. There seems to be no functional implication.

**Comment by marcelveldt** [comment]
Fix released in 1.1.17

**Comment by erkr** [comment]
Thanks!
